samples/scalers/cron/standard_v1alpha2_autoscaler.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: standard.oam.dev/v1alpha1
```

Comment:
A plain autoscaler CR is not enough to prove this trait works.

Should include a full OAM sample, and the trait should be tested against:

1. (Must have) A abstract workload such as `Containerized` (will be named as `PodScpecableWorkload` soon) in kubevela.
2. (Must have) A plain built-in workload such as `Deployment`.
3. (Optional) A custom workload such as `CloneSet`.

For fast development we didn't require automatic e2e test suites but manual e2e test (i.e. self-proven yamls) is MUST have.

Reply:
Totally agree with @resouer , we MUST give a full OAM sample instead of the plain autoscaler CR sample.

Reply:
> 1. (Must have) A abstract workload such as `Containerized` (will be named as `PodSpecWorkload` soon) in kubevela.
> 2. (Must have) A plain built-in workload such as `Deployment`.
> 3. (Optional) A custom workload such as `CloneSet`.

1. supported.  Example manifests in folder `samples/autoscalers/cron/PodSpecWorkload`
2. In progress
3. Pending

Reply:
2. Supported Deployment. Example manifests in `samples/autoscalers/cron/deployment`.

But for both `PodSpecWorkload` and `Deployment`, there are warnings in ApplicationConfiguration events as I set ownerRef of workloads (child resource `Deployment` of `PodSpecWorkload`, and `Deployment` itself) to AutoScaler trait. I haven't figured out how to fix it.

- PodSpecWorkload
```
  Warning  cannot apply the deployment  31m                   PodSpecWorkload                            
Deployment.apps "component-scaler" is invalid: metadata.ownerReferences: Invalid value: 
[]v1.OwnerReference{v1.OwnerReference{APIVersion:"standard.oam.dev/v1alpha1", Kind:"PodSpecWorkload", 
Name:"component-scaler", UID:"fbab3f47-7f23-46ab-abdf-98a78513d9d5", Controller:(*bool)(0xc008ed6ab0), 
BlockOwnerDeletion:(*bool)(0xc008ed6ab1)}, v1.OwnerReference{APIVersion:"standard.oam.dev/v1alpha1", 
Kind:"Autoscaler", Name:"trait-scaler", UID:"fd168b3b-a70d-4edf-8d81-525af31037c0", Controller:(*bool)
(0xc008ed6ab2), BlockOwnerDeletion:(*bool)(0xc008ed6ab3)}}: Only one reference can have Controller set to true. 
Found "true" in references for component-scaler and trait-scaler

```

- Deployment
```
cannot apply workload "component-scaler": existing object is not controlled by UID "54cb9f2a-98e3-4f85-8f1d-47522035533e"
```

Reply:
You can't let controller A and B control another resource at same time. The trait controller should "take over" in this case. I am not sure if it should "give back" after scaling finished, manual scaler trait or @ryanzhang-oss may have more input here.